driver/src/main/java/org/neo4j/driver/internal/messaging/BoltProtocolVersion.java

```diff
@@ -55,6 +55,21 @@ public int toInt()
         return shiftedMinor | majorVersion;
     }
 
+    public int toIntRange(BoltProtocolVersion minVersion)
+    {
+        if(majorVersion != minVersion.majorVersion)
+        {
+            throw new IllegalArgumentException( "Versions should be from the same version" );
+        }
+        else if (minorVersion < minVersion.minorVersion)
+        {
+            throw new IllegalArgumentException("Max version should be newest than min version");
```

Comment:
nit: I know its less consistently applied in the driver but can you format the changes using the server checkstyles

Reply:
I've did a format on the file, what's off in the current format?

Reply:
the space before and after the bracket : `throw new IllegalArgumentException( "Max version should be newest than min version" );`